.github/workflows/update-repo-request-template.yml

```diff
@@ -55,20 +63,21 @@ jobs:
       - name: Display File
         if: ${{ steps.change-detection.outputs.changed == 'true' }}
         run: cat $ISSUE_TEMPLATE_FILE
-      - name: 'Yamllint'
+      - name: "Yamllint"
         if: ${{ steps.change-detection.outputs.changed == 'true' }}
         uses: karancode/yamllint-github-action@master
         with:
           yamllint_file_or_dir: ${{ env.ISSUE_TEMPLATE_FILE }}
           yamllint_strict: true
-      - name: Commit Changes          
+      - name: Commit Changes
         if: ${{ steps.change-detection.outputs.changed == 'true' }}
-        env: 
+        env:
+          CI_COMMIT_USER: "xpirit-training-bot[bot]"
+          CI_COMMIT_USER_MAIL: 139118093+xpirit-training-bot[bot]@users.noreply.github.com
           CI_COMMIT_MESSAGE: "chore(org): Update Issue Template File"
-          CI_COMMIT_AUTHOR: Continuous Integration
         run: |
-          git config --global user.name "xpirit-training-bot[bot]"
```

Comment:
Please put the git user and git email into a repo variable :) same for the other workflow file as well.

Reply:
done